atom/renderer/api/atom_api_web_frame.cc

```diff
@@ -75,7 +77,15 @@ void WebFrame::RegisterElementResizeCallback(
 }
 
 void WebFrame::AttachGuest(int id) {
+  // This is a workaround for a strange issue on windows with background tabs
+  // libchromiumcontent doesn't appear to be making the check for
+  // params.dispostion == NEW_BACKGROUND_TAB in WebContentsImpl
+  // This results in the BrowserPluginGuest trying to access the native
+  // window before it's actually ready. This hack works around that
+  // by always delaying the access
+  content::BrowserPluginManager::Get()->GetBrowserPlugin(id)->updateVisibility(false);
```

Comment:
Wow this was a strange one, nice job.  I don't think I've ever run into something like this but I've always compiled all of my own source on my projects and not used prebuilt stuff.